ssl_test.go

```diff
@@ -171,9 +198,13 @@ func assertConnectionSslOk(t testing.TB, serverOpts, clientOpts SslOpts) {
 func assertConnectionTntFail(t testing.TB, serverOpts, clientOpts SslOpts) {
 	t.Helper()
 
-	inst, err := serverTnt(serverOpts, clientOpts, AutoAuth)
-	serverTntStop(inst)
+	inst, err := serverTnt(serverOpts, AutoAuth)
+	if err != nil {
+		t.Errorf("An unexpected server error %q", err.Error())
+	}
+	defer serverTntStop(inst)
```

Comment:
1. `test_helpers.StopTarantoolWithCleanup()` cleanups data directories, so it would be nice to call it anyway.
2. If we don't want to resume the test then it is better to use `Fatalf`.
```suggestion
	defer serverTntStop(inst)
	if err != nil {
		t.Fatalf("An unexpected server error %q", err.Error())
	}
```

Reply:
Reworked